### DISCUSSION TITLE
what does 'npx tailwindcss init -p' do?

### DISCUSSION THREAD

**Discussion by Seongil-Shin** [tailwindlabs/tailwindcss#6773] [Help] [answered]
I followed this [guide](https://tailwindcss.com/docs/guides/create-react-app) on react app created by create-react-app.

After that, i could hot-reload even when i change tailwind-css codes. But i read some articles that says we need carco and some complicate configs on 'package.json' to hot-reload tailwindCSS. But i just followed the guide and i could make it.

I guess the command 'npx tailwindcss init -p' on the guide may config create-react-app so make it possible to hot-reload. However i don't know exactly.  

Anyway, i'd want to ask what 'npx tailwindcss init -p' does? and how the guide make it possible to hot-reload?

**Comment by olivermrose** [comment]
All it does is create a boilerplate `postcss.config.js` file.

**Reply by Seongil-Shin** [reply]
Thank you for answer!

**Comment by adamwathan** [comment, marked as answer]
With create-react-app v5 you don't need CRACO or any other workarounds anymore.  The `-p` just scaffolds a basic `postcss.config.js` file like @xiBread mentioned 👍🏻

**Reply by Seongil-Shin** [reply]
Thank you for answer!
